### PR TITLE
Details for timeout errors

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/shared/core/BackgroundScriptUpdateHandler.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export class BackgroundScriptUpdateHandler {
     const number = getRandomInteger();
     return Promise.race([
       sendPortMessage<{ syn: number }, { ack: number }>(port, { syn: number }),
-      rejectAfterDelay(1000),
+      rejectAfterDelay(1000, 'sendPortMessage'),
```
Comment (zerts) [inline src/shared/core/BackgroundScriptUpdateHandler.ts:78]: I suggest adding user-readable errors everywhere. At least, error screen will look more friendly:))

Comment (everdimension) [reply src/shared/core/BackgroundScriptUpdateHandler.ts:78]: I think it's quite hard to come up with "user readable" messages for each request. What would you suggest?

Comment (zerts) [reply src/shared/core/BackgroundScriptUpdateHandler.ts:78]: Background port is unavailable

**Review by zerts — APPROVED** [review]

File: src/shared/core/BackgroundScriptUpdateHandler.ts (hunk 1 above)
Comment (everdimension) [reply src/shared/core/BackgroundScriptUpdateHandler.ts:78]: I am not sure this is more user readable 😁

**Pull request merged**